src/components/PoolingWidget/AddFunding.tsx

```diff
@@ -24,7 +24,7 @@ const AddFunding: React.FC<AddFundingProps> = ({ txIdentifier, txReceipt }) => (
         </strong>
         <p>
           {txReceipt
-            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions above.'
+            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions bellow.'
```

Comment:
```suggestion
            ? 'Your new liquidity has been successfully mined and submitted! Please carefully read the instructions below.'
```

bellow means to scream, or yell out

Reply:
Of course I meant that :D

Reply:
Also:

![](https://i.pinimg.com/originals/50/8d/b3/508db30049a6b33204ea80858e95066a.jpg)

Reply:
"FRIENDS ONLY" - show the whole picture bro, fake news